_funcutil.ts

```diff
@@ -3,7 +3,8 @@ import { inspect } from "./_inspect.ts";
 /**
  * Rewrite the function name.
  */
-export function rewriteName<F extends (...args: unknown[]) => unknown>(
+// deno-lint-ignore no-explicit-any
+export function rewriteName<F extends (...args: any[]) => unknown>(
```

Comment:
Wouldn't it be better to use `Fn` here too?

Reply:
But since `_funcutil.ts` depends on `_annotation.ts`, their pros and cons are pretty much the same.